examples/show_util.py

```diff
@@ -0,0 +1,12 @@
+"""
+Demonstrates show utility
+"""
+
+import pygfx as gfx
+
+material = gfx.MeshPhongMaterial()
+geometry = gfx.box_geometry(100, 100, 100)
+cube = gfx.Mesh(geometry, material)
```

Comment:
I've grown to like this notation, and its a wee bit code ;)
```suggestion
cube = gfx.Mesh(
    gfx.box_geometry(100, 100, 100),
    gfx.MeshPhongMaterial(),
)
```

Reply:
But this is personal, so feel free to ignore.